mithril-aggregator/src/database/provider/signer.rs

```diff
@@ -11,20 +13,26 @@ use mithril_common::StdResult;
 
 use crate::signer_registerer::SignerRecorder;
 
+#[cfg(test)]
+use mockall::automock;
+
 /// Signer record is the representation of a stored signer.
 #[derive(Debug, PartialEq, Clone)]
 pub struct SignerRecord {
     /// Signer id.
-    signer_id: String,
+    pub signer_id: String,
 
     /// Pool ticker of the signer.
-    pool_ticker: Option<String>,
+    pub pool_ticker: Option<String>,
 
     /// Date and time when the signer was created.
-    created_at: DateTime<Utc>,
+    pub created_at: DateTime<Utc>,
 
     /// Date and time when the signer was updated.
-    updated_at: DateTime<Utc>,
+    pub updated_at: DateTime<Utc>,
+
+    /// Date and time when the signer first registered.
```

Comment:
This is probably the last registration date?
```suggestion
    /// Date and time when the signer last registered.
```